### ISSUE TITLE
Is it possibile to support also strm32F1xx ?

### ISSUE THREAD

**Issue by massimosala** [micropython/stm32lib#24]
Hi

I am a C programmer but not expert on porting micropython to new micros.

I read the instructions and extract the library files from the last version at ST github.
Please see the attached ZIP archive.

I have a STM32-VL-Discovery development board, with a STM32F100RBT6B.

I offer myself as beta-tester.

Thanks, Massimo
[STM32F1xx.zip](https://github.com/micropython/stm32lib/files/11347325/STM32F1xx.zip)
[stm32vldiscovery-stm32-value-line-discovery-stmicroelectronics.pdf](https://github.com/micropython/stm32lib/files/11347330/stm32vldiscovery-stm32-value-line-discovery-stmicroelectronics.pdf
[stm32f100rb.pdf](https://github.com/micropython/stm32lib/files/11347333/stm32f100rb.pdf)
)

**Comment by dpgeorge** [comment]
Yes it's possible to support the STM32F1xx series, but I don't think it's worth doing, because they are old MCUs.

See this comment here, which discusses STM32F3xx: https://github.com/micropython/micropython/pull/6955#issuecomment-1326938967
